### ISSUE TITLE
Test failure(s): HueLightHandlerOSGiTest

### ISSUE THREAD

**Issue by sjka** [eclipse/smarthome#1751]
I ran into the following errors where the properties have not been set as expected.

Are they potentially related to #1652 ?

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.461 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
assert no discovery result for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.193 sec
assert correct thing UID(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.007 sec
assert no thing UID for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.001 sec
assert correct supported types(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.008 sec
assert valid DiscoveryResult(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.014 sec
Running org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.464 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
verify offline is set without bridge offline status(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.147 sec
assert that HueBridgeHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.104 sec
assert that an existing user is used if authentication was successful(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.045 sec
assert correct status if authentication failed for old user(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.046 sec
verify status if link button is not pressed(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.05 sec
assert that a new user is added to config if not existing yet(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.03 sec
verify status if new user cannot be created(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.028 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.143 sec - in org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
assert hue light registration(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 0.061 sec
assert startSearch is called(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 1.072 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
Tests run: 32, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 40.518 sec <<< FAILURE! - in org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
assert command for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.195 sec
assert command for color channel: black(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.112 sec
assert command for color channel: white(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.092 sec
assert command for color channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.29 sec
assert command for osram par16 50 for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.099 sec
assert command for color channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.178 sec
assert command for osram par16 50 for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.115 sec
assert that HueLightHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.019 sec
assert command for osram par16 50 for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.08 sec
assert percentage value of color temperature when ct: 153(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.013 sec
assert percentage value of color temperature when ct: 326(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.005 sec
assert percentage value of color temperature when ct: 500(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.004 sec
assert command for color temperature channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.092 sec
assert command for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.088 sec
assert command for color channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.084 sec
assert command for color channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.091 sec
assert command for brightness channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.086 sec
assert command for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.079 sec
assert command for color temperature channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.024 sec  <<< FAILURE!
java.lang.AssertionError:
Expected: is "LCT001"
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest$_assertSendCommand_closure15.doCall(HueLightHandlerOSGiTest.groovy:545)
	at sun.reflect.GeneratedMethodAccessor81.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure.call(Closure.java:439)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.with(DefaultGroovyMethods.java:235)
	at org.codehaus.groovy.runtime.dgm$614.invoke(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoMetaMethodSiteNoUnwrapNoCoerce.invoke(PojoMetaMethodSite.java:271)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:53)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.assertSendCommand(HueLightHandlerOSGiTest.groovy:544)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.assertSendCommand(HueLightHandlerOSGiTest.groovy)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.this$3$assertSendCommand(HueLightHandlerOSGiTest.groovy)

assert command for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.12 sec
assert command for color channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.091 sec
assert command for color channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.114 sec
assert command for color channel: red(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.083 sec
assert command for brightness channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.31 sec
assert command for alert channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.281 sec
assert command for brightness channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.152 sec
assert command for color temperature channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.078 sec
assert that HueLightHandler status detail is set to bridge offline when the bridge is offline(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.041 sec
assert command for osram par16 50 for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.091 sec
assert command for color channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.075 sec
assert command for color channel: blue(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.079 sec
assert command for effect channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.231 sec

Results :

Failed tests:
  HueLightHandlerOSGiTest.assert command for color temperature channel: 50%:274->this$3$assertSendCommandForColorTemp:-1->assertSendCommandForColorTemp:488->this$3$assertSendCommand:-1->assertSendCommand:-1->assertSendCommand:544
Expected: is "LCT001"
     but: was null

Tests run: 46, Failures: 1, Errors: 0, Skipped: 0
```

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.44 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest
assert no discovery result for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.196 sec
assert correct thing UID(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.007 sec
assert no thing UID for unknown device(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.001 sec
assert correct supported types(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.008 sec
assert valid DiscoveryResult(org.eclipse.smarthome.binding.hue.test.HueBridgeDiscoveryParticipantOSGITest)  Time elapsed: 0.013 sec
Running org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.439 sec - in org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest
verify offline is set without bridge offline status(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.132 sec
assert that HueBridgeHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.102 sec
assert that an existing user is used if authentication was successful(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.044 sec
assert correct status if authentication failed for old user(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.057 sec
verify status if link button is not pressed(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.031 sec
assert that a new user is added to config if not existing yet(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.03 sec
verify status if new user cannot be created(org.eclipse.smarthome.binding.hue.test.HueBridgeHandlerOSGiTest)  Time elapsed: 0.028 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.153 sec - in org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest
assert hue light registration(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 0.065 sec
assert startSearch is called(org.eclipse.smarthome.binding.hue.test.HueLightDiscoveryServiceOSGITest)  Time elapsed: 1.079 sec
Running org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
Tests run: 32, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 39.505 sec <<< FAILURE! - in org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest
assert command for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.152 sec
assert command for color channel: black(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.105 sec
assert command for color channel: white(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.104 sec
assert command for color channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.275 sec
assert command for osram par16 50 for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.084 sec
assert command for color channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.19 sec
assert command for osram par16 50 for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.11 sec
assert that HueLightHandler is registered and unregistered(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.02 sec
assert command for osram par16 50 for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.105 sec
assert percentage value of color temperature when ct: 153(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.013 sec
assert percentage value of color temperature when ct: 326(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.005 sec
assert percentage value of color temperature when ct: 500(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 0.005 sec
assert command for color temperature channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.093 sec
assert command for color temperature channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.107 sec
assert command for color channel: 0%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.091 sec
assert command for color channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.129 sec
assert command for brightness channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.129 sec
assert command for brightness channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.082 sec
assert command for color temperature channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.108 sec
assert command for color temperature channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.097 sec
assert command for color channel: 50%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.089 sec
assert command for color channel: off(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.099 sec
assert command for color channel: red(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.082 sec
assert command for brightness channel: increase(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.104 sec  <<< FAILURE!
java.lang.AssertionError:
Expected: is "LCT001"
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
	at org.junit.Assert$assertThat.callStatic(Unknown Source)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest$_assertSendCommand_closure15.doCall(HueLightHandlerOSGiTest.groovy:545)
	at sun.reflect.GeneratedMethodAccessor81.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure.call(Closure.java:439)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.with(DefaultGroovyMethods.java:235)
	at org.codehaus.groovy.runtime.dgm$614.invoke(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoMetaMethodSiteNoUnwrapNoCoerce.invoke(PojoMetaMethodSite.java:271)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:53)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.assertSendCommand(HueLightHandlerOSGiTest.groovy:544)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.assertSendCommand(HueLightHandlerOSGiTest.groovy)
	at org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest.this$3$assertSendCommand(HueLightHandlerOSGiTest.groovy)

assert command for alert channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 3.3 sec
assert command for brightness channel: decrease(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.195 sec
assert command for color temperature channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.083 sec
assert that HueLightHandler status detail is set to bridge offline when the bridge is offline(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.037 sec
assert command for osram par16 50 for brightness channel: on(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.088 sec
assert command for color channel: 100%(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.082 sec
assert command for color channel: blue(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 1.125 sec
assert command for effect channel(org.eclipse.smarthome.binding.hue.test.HueLightHandlerOSGiTest)  Time elapsed: 2.191 sec

Results :

Failed tests:
  HueLightHandlerOSGiTest.assert command for brightness channel: increase:417->this$3$assertSendCommandForBrightness:-1->assertSendCommandForBrightness:497->this$3$assertSendCommand:-1->assertSendCommand:-1->assertSendCommand:544
Expected: is "LCT001"
     but: was null

Tests run: 46, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by sjka** [comment]
as we were just talking about why hudsons didn't fail: my hit rate is about .3:

```
iteration 0 -> 0
iteration 1 -> 0
iteration 2 -> 0
iteration 3 -> 1
iteration 4 -> 1
iteration 5 -> 0
iteration 6 -> 1
iteration 7 -> 0
iteration 8 -> 0
iteration 9 -> 0
```

**Comment by maggu2810** [comment]
[32.log.txt](https://github.com/eclipse/smarthome/files/336456/32.log.txt)

**Comment by afuechsel** [comment]
should be fixed with #1799. please recheck @sjka

**Comment by sjka** [comment]
running 40 iterations didn't bring up a single failure, so I think it's safe to say this indeed has been fixed.
Thanks!
